CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+**Breaking changes**
+
+### Renaming of `*Options` and `*Props` to `*Config`
+
+We had several exports where the configuration passed to constructs or functions was using a class with a different suffix then `Config`. If you are using a language like C#, Java, Python, or Go you might have needed to specify these classes in your CDKTF application. To simplify the usage we changed every suffix to be `Config`, so if you currently use one of the others please rename them to use the `Config` suffix. No options were changed in the process, only the names were aligned.
```

Comment:
I think an example for each of them (Options and Props) at the end of this section could help a lot – doesn't have to be language specific, just something to showcase. An example will also catch the eye easier while skimming the changelog 😄